shortex.tex

```diff
@@ -1,188 +1,622 @@
+includes some of these packages, we recommend editing those style files to remove the package imports.
+
+The ShorTeX package has a few options:
+\bdesc
+\item[\texttt{manualnumbering}] Do not include \texttt{autonum.sty}. This disables automatic equation numbering.
+\item[\texttt{blackhypersetup}] Switch hyperlinks, citations, references, etc.~to be typeset in black font. The default is dark blue.
+\edesc
+\textbf{You must compile your document 4 times when using ShorTeX} to ensure that equation
+numbers and references update properly.
+
+\section{Shorthands for existing commands}
+
+\subsection{Environments}
+
+\LaTeX~documents often includes a lot of verbose code
+related to creating environments (\verb!\begin{blah}...\end{blah}!). ShorTeX provides a set of 
+shortened macros for common environments.
+Note that all theorem-like environments (theorem, lemma, proposition, etc.) 
+are numbered by default; unnumbered versions can be obtained by appending a \verb!u!. For example,
+\verb!\bthmu...\ethmu! creates an unnumbered theorem environment, while
+\verb!\blemu...\elemu! creates an unnumbered lemma environment.
+
+\bcent
+\btabr{@{}ll@{}}
 \toprule
-Description 				& Example					& Text style 				& Display style \\ \midrule
-Round brackets (i.e., parentheses)	& \verb!\rbra{\frac{x}{y}}!        	& $\rbra{\frac{x}{y}}$ 		& $\displaystyle\rbra{\frac{x}{y}}$ \\[10pt]
-Curly brackets 			& \verb!\cbra*{\frac{x}{y}}!    	& $\cbra*{\frac{x}{y}}$ 	& $\displaystyle\cbra*{\frac{x}{y}}$ \\[10pt]
-Square brackets 			& \verb!\sbra[\bigg]{\frac{x}{y}}!        	& $\sbra[\bigg]{\frac{x}{y}}$ 	& $\displaystyle\sbra[\bigg]{\frac{x}{y}}$ \\[10pt]
+Environment & Syntax \\ \midrule
+abstract & \verb!\babs...\eabs!\\ \midrule
+itemize & \verb!\bitems...\eitems!\\
+enumerate & \verb!\benum...\eenum!\\
+description & \verb!\bdesc...\edesc!\\ \midrule
+algorithm & \verb!\balg...\ealg!\\
+algorithmic & \verb!\balgc...\ealgc!\\ \midrule
+table & \verb!\btab...\etab!\\
+subtable & \verb!\bsubtab...\esubtab!\\
+tabular & \verb!\btabr...\etabr!\\ \midrule
+figure & \verb!\bfig...\efig!\\
+figure* & \verb!\bfigs...\efigs!\\
+subfigure & \verb!\bsubfig...\esubfig!\\ \midrule
+center & \verb!\bcent...\ecent!\\ \midrule
+align & \verb!\[...\]!\\ 
+inline math & \verb!$...$!\\ \midrule
+\multicolumn{2}{c}{\emph{Note: These are numbered theorem-like environments.}}\\
+\multicolumn{2}{c}{\emph{For unnumbered, append a \texttt{u}: e.g.,} \texttt{bthmu...ethmu}.}\\
+theorem & \verb!\bthm...\ethm!\\ 
+lemma & \verb!\blem...\elem!\\
+proposition & \verb!\bprop...\eprop!\\
+corollary & \verb!\bcor...\ecor!\\
+conjecture & \verb!\bconj...\econj!\\
+definition & \verb!\bdef...\edef!\\
+assumption & \verb!\bassump...\eassump!\\
+example & \verb!\bexa...\eexa!\\
+remark & \verb!\brmk...\ermk!\\
+fact & \verb!\bfact...\efact!\\
+exercise & \verb!\bexer...\eexer!\\
+proof & \verb!\bprf...\eprf!\\
```

Comment:
proof environments should be in a separate row since they aren't theorem-like

Reply:
added midrule